Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -353,6 +353,18 @@ FullScreenEnabled:
     WebCore:
       default: false
 
+FullySuspendBackgroundTabs:
```

Comment:
I find this naming confusing. Maybe it should be about taking Suspended assertions or not?
E.g. "ShouldTakeSuspendedAssertions" and make it true by default?